libBuild.sh

```diff
@@ -219,6 +225,10 @@ function publish_layer {
     then compat_list=("provided" "provided.al2" "provided.al2023" "dotnetcore3.1" "dotnet6")
```

Comment:
@chaudharysaket -- Do you know if this is still correct? It seems like it should fail, since .NET Core 3.1 is no longer supported, but the Extension was built fairly recently.

Reply:
AWS compatibility check allows previously supported layer, example `python3.6` still works in the compat_list.
We need to update the compat_list for extension layers.  Since we will start supporting `dotnet6` & `dotnet8` with a dedicated layer, we shall update the compat_list for extension to reflect the same.

Reply:
Got it, thanks!